languageserver/test/index.test.ts

```diff
@@ -309,7 +308,7 @@ describe("diagnostics", () => {
         }
 
         resolve(docsNotifications)
-      })
+      }, true)
```

Comment:
What is the magic true here?

Reply:
this magic true made me spent probably hours combined :D I will refactor it now -.- every time I forget to add it, what it does is enables the flowClient integration which is needed for new import syntax.

Reply:
Will do that part of another PR tho, a bit busy now with release sorry.